### PR TITLE
Suez Water sensor documentation

### PR DESCRIPTION
**Pull request by ooii**
**Description:**
This is documentation for the new Suez Water sensor. 
It describes how the sensor should be used and what information it provides.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23844

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/suez.markdown
Comment (klaasnicolaas) [inline source/_components/suez.markdown]: Please remove the `redirect_from`

File: source/_components/suez.markdown
Comment (klaasnicolaas) [inline source/_components/suez.markdown:6]:
```suggestion
ha_category:
  - Water
  - Sensor
```

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_components/suez.markdown
Comment (ooii) [reply source/_components/suez.markdown]: Done

File: source/_components/suez.markdown
Comment (fabaff) [inline source/_components/suez.markdown]: Remove this. It would create an additional category with one entry.

Comment (ooii) [reply source/_components/suez.markdown]: Done

**Review by frenck — DISMISSED** [review]

**Comment by klaasnicolaas** [comment]
✅ Can be merged after merging the parent PR 👍

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @ooii! Good job! 👍

**Pull request merged**
